src/lib/operators/operator_performance_data.hpp

```diff
@@ -13,7 +13,10 @@ namespace opossum {
 struct OperatorPerformanceData : public Noncopyable {
   virtual ~OperatorPerformanceData() = default;
 
+  bool executed{false};
   std::chrono::nanoseconds walltime{0};
+  uint64_t output_row_count{0};
```

Comment:
Wondering if there is value in using std::optional to make it explicit that some operators do not have an output?

Reply:
That optional would also have to cover output_chunk_count. I added an additional flag:

```
  // Some operators do not return a table (e.g., Insert).
  // Note: The operator returning an empty table will be expressed as has_output == true, output_row_count == 0
  bool has_output{false};
  uint64_t output_row_count{0};
  uint64_t output_chunk_count{0};
```